allennlp_models/classification/models/transformer_classification_tt.py

```diff
@@ -0,0 +1,103 @@
+import logging
+from typing import Dict, Optional
+
+import torch
+from allennlp.data import Vocabulary
+from allennlp.models import Model
+from allennlp.modules.transformer import TransformerEmbeddings, TransformerStack, TransformerPooler
+from torch.nn import Dropout
+
+logger = logging.getLogger(__name__)
+
+
+@Model.register("transformer_classification_tt")
+class TransformerClassificationTransformerToolkit(Model):
```

Comment:
That's a verbose name! :) Is there a reason to name models that use the transformer toolkit thus?

Reply:
I did it this way for the "MC" models, and I wanted to stay consistent. But "MC" is much shorter than "Classification", so now it looks silly.

We could rename all the Transformer Toolkit models to just "TT"? Then we also don't have to say "Transformer" twice.